### PR TITLE
Lifts many loadout restrictions, adds some new items to loadout

### PR DESCRIPTION
**Pull request by TheTrollDoctor**

### CODE REVIEW

File: maps/torch/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
 
 /datum/gear/accessory/tags
 	display_name = "dog tags"
 	path = /obj/item/clothing/accessory/badge/solgov/tags
 
-/datum/gear/accessory/torch_patch
-	display_name = "Torch mission patch"
-	path = /obj/item/clothing/accessory/solgov/torch_patch
+///datum/gear/accessory/torch_patch
+//	display_name = "Torch mission patch"
```
Comment (MalfunctioningAI) [inline maps/torch/loadout/loadout_accessories.dm]: Perhaps keep this for fluff reasons, and rename it to "old ship patch" or similiarly?

File: maps/torch/loadout/loadout_accessories.dm [hunk 2]
```diff
@@ -1,78 +1,78 @@
...
+//	path = /obj/item/clothing/accessory/solgov/torch_patch
 
-/datum/gear/accessory/pilot_pin
-	display_name = "pilot's qualification pin"
-	path = /obj/item/clothing/accessory/solgov/speciality/pilot
-	allowed_roles = list(/datum/job/captain, /datum/job/hop, /datum/job/bridgeofficer, /datum/job/pathfinder)
+///datum/gear/accessory/pilot_pin
+//	display_name = "pilot's qualification pin"
```
Comment (MalfunctioningAI) [inline maps/torch/loadout/loadout_accessories.dm]: Why remove it?

**Comment by MalfunctioningAI** [comment]
There's plenty of commented out code in this PR. Why?

**Comment by MalfunctioningAI** [comment]
>6 incorrect indentations (expecting exactly 0)

>\## UNIT_TEST ##: /datum/gear/accessory/chaps - /obj/item/clothing/accessory/chaps/black ('black chaps'): Did not find a gear_tweak's icon_state 'chaps_black' in the icon 'icons/obj/clothing/ties.dmi'.
>\## UNIT_TEST ##: /datum/gear/accessory/chaps - /obj/item/clothing/accessory/chaps ('brown chaps'): Did not find a gear_tweak's icon_state 'chaps' in the icon 'icons/obj/clothing/ties.dmi'.
>\## UNIT_TEST ##: /datum/gear/head/cowboys - /obj/item/clothing/head/cowboy_hat/wide_black ('black cowboy hat'): Did not find a gear_tweak's icon_state 'cowboy_black' in the icon 'icons/obj/clothing/hats.dmi'.
>\## UNIT_TEST ##: /datum/gear/head/cowboys - /obj/item/clothing/head/cowboy_hat/wide_brown ('brown cowboy hat'): Did not find a gear_tweak's icon_state 'cowboy_brown' in the icon 'icons/obj/clothing/hats.dmi'.
>\## UNIT_TEST ##: !!! FAILURE !!! [LOADOUT: Entries shall have valid icon states]: One or more /datum/gear definitions had paths with invalid icon states.

**Comment by MalfunctioningAI** [comment]
I put the icons for the cowboy hats in the indiebay hats .dmi, for your information. Since you changed them to the hats.dm file, you have to move the icons in the .dmi files to the hats.dmi file.

File: code/modules/client/preference_setup/loadout/lists/accessories.dm [hunk 3]
```diff
@@ -66,6 +66,17 @@
 	ntawards["command medal"] = /obj/item/clothing/accessory/medal/gold/nanotrasen
 	gear_tweaks += new/datum/gear_tweak/path(ntawards)
 
+/datum/gear/accessory/chaps
```
Comment (MalfunctioningAI) [inline code/modules/client/preference_setup/loadout/lists/accessories.dm]: This means it's a tie subclass, fairly sure.

Comment (TheTrollDoctor) [reply code/modules/client/preference_setup/loadout/lists/accessories.dm]: ALL accessories are tie subclasses.

**Comment by TheTrollDoctor** [comment]
Code's commented out so it's unavailable until we get the code in to restrict loadout by branch. Most of the commented stuff is to document bits that need to be implemented as branch-specific when we get that code.

File: maps/torch/loadout/loadout_accessories.dm (hunk 1 above)
Comment (TheTrollDoctor) [reply maps/torch/loadout/loadout_accessories.dm]: In fluff, this isn't the Torch - it's a ship of the same model that was renovated in a scrapheap.

File: code/modules/client/preference_setup/loadout/lists/accessories.dm (hunk 3 above)
Comment (TheTrollDoctor) [reply code/modules/client/preference_setup/loadout/lists/accessories.dm]: Ignore my deleted comment. I'll fix this.

Comment (TheTrollDoctor) [reply code/modules/client/preference_setup/loadout/lists/accessories.dm]: Nope. It's defined as a clothing accessory in code\modules\clothing\under\accessories\clothing

File: maps/torch/loadout/loadout_accessories.dm (hunk 1 above)
Comment (MalfunctioningAI) [reply maps/torch/loadout/loadout_accessories.dm]: Ah, ok! :+1:

**Comment by TheTrollDoctor** [comment]
In-game issues have been resolved.

**Comment by MalfunctioningAI** [comment]
waiting for travis bby

**Comment by MalfunctioningAI** [comment]
xaxaxaxa

**Pull request merged**